package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "test": "NODE_ENV=test mocha -R spec ./test/* --require babel-polyfill --require babel-register",
+    "test": "PORT=3003 NODE_ENV=test mocha -R spec ./test/* --require babel-polyfill --require babel-register",
```

Comment:
I do not think it's a good idea to override the port variable in package.json. Can we but this in https://github.com/france-connect/service-provider-example/blob/master/.circleci/config.yml#L37 instead ? In that way, we will be able to run the test in different environments

Reply:
The good trick to do that in test, it is that you can play test event if your server is up which is impossible now.
We can let the test in circleci do it normally.